dev/benchmarks/macrobenchmarks/test/frame_policy.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:developer';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:e2e/e2e.dart';
+
+import 'package:macrobenchmarks/src/simple_scroll.dart';
+
+void main() {
+  final E2EWidgetsFlutterBinding binding =
+      E2EWidgetsFlutterBinding.ensureInitialized() as E2EWidgetsFlutterBinding;
+  testWidgets(
+    'Frame Counter and Input Delay for benchmarkLive',
+    (WidgetTester tester) async {
+      await tester.pumpWidget(MaterialApp(home: Scaffold(body: SimpleScroll())));
```

Comment:
By doing so I'm avoiding extra operation to select the page. It might be a good idea to do the same for other pages with E2E test version, or to put this together with the util code in #61509